plugin/.env.example

```
@@ -1,2 +1,4 @@
-AXIOM_API_TOKEN=""
-AXIOM_DATASET_NAME=""
\ No newline at end of file
+AXIOM_API_TOKEN=
+AXIOM_DATASET_NAME=
+HTTP_AUTH_TOKEN=
+HTTP_BASE_URL=
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)